src/index.js

```diff
@@ -40,16 +40,22 @@ export class Conrec {
     this.hasMinMax = false;
   }
 
+  /**
+   * @typedef {Object} Output
+   * @property {any} contours
+   * @property {boolean} timeout - Whether contour generation finishes in the specified timeout or not
```

Comment:
Doesn't it return the opposite?

Reply:
I need just to negate the statement

Reply:
I'd write something like "Whether contour generation had to stop early because it reached the timeout".